.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+## PR Description: *short_description*
+
+### Tickets:
+ - *ticket-id*
+
+### Details:
+ - *item1*
+ - *...*
+
+### Pull Request Readiness Checklist
+
+See details at https://github.com/openvinotoolkit/openvino/wiki/Contribute#making-a-good-pull-request
+
+- [ ] I agree to contribute to the project under Apache 2 License.
+- [ ] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenVINO
+- [ ] The PR is proposed to proper branch
+- [ ] There is reference to original bug report and related work
+- [ ] Code changes:
+    - [ ] Comments
+    - [ ] Code style
+- [ ] Validation:
+    - [ ] Unit tests
+    - [ ] Function tests
+    - [ ] e2e tests
+    - [ ] Component dependent tests
+- [ ] Documentation:
+    - [ ] User guide update
+    - [ ] Developer guide update
+    - [ ] Specification update
+    - [ ] Component specific guides
```

Comment:
IMHO, if the PR relates to the Documentation it will be mentioned in its Title/Details section. I don't see the reason to duplicate this information in the Checklist.

Reply:
If you add a new functionality you need to add a documentation for it. In this checklist contributor can think about different documentations and update if it is required.

Reply:
For me the checklist looks like obligatory conditions. But developer don't need to update `User guide` if the PR modifies internal things.